src/AutoRest.CSharp/Common/Output/Models/FieldDeclaration.cs

```diff
@@ -23,13 +24,24 @@ public FieldDeclaration(FieldModifiers modifiers, CSharpType type, string name,
                   writeAsProperty: writeAsProperty)
         { }
 
-        public FieldDeclaration(FieldModifiers modifiers, CSharpType type, string name, FormattableString? defaultValue, SerializationFormat serializationFormat, bool writeAsProperty = false)
+        public FieldDeclaration(FormattableString description, FieldModifiers modifiers, CSharpType type, string name, ValueExpression? initializationValue = null)
+            : this(Description: description,
```

Comment:
these explicit naming seems redundant, most of them are not optional

Reply:
well I intentional add them to make those parameters more clear - especially when a method consumes a lot of parameters, and you might have a few arguments with string types. It is very confusing to have somehting like this:
```
DoSomething("a", "b", "c")
```
or
```
DoSomething("a", null, null)
```
it is very likely to go wrong when we add a new required parameter (when we have to) or add a new optional parameter, if it happens to have the same type, the compiler never shows an error for you